app/models/friendship.rb

```diff
@@ -1,11 +1,55 @@
+class SelfFriendshipValidator < ActiveModel::Validator
```

Comment:
**Kindly work on this issue:**
- When i send friend requests i can accept and decline these friend requests by myself instead of every user.

![Screen Shot 2021-09-20 at 14 11 35](https://user-images.githubusercontent.com/22258274/134060837-b01c74a9-3cc5-43e8-8529-fe089bb14140.png)

Reply:
- [ ] I am still able to accept a request that I sent myself to another user, this should not happen, please fix it to comply with the project requirements. (For example, if I am Elias and I send a friendship request to Carlos, I am able to accept the friendship request by myself and force a friendship in the app)